script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt

```diff
@@ -664,32 +754,35 @@ I told you not to leave the room, so why were you outside meeting with Arcueid?
 [sha:b997f4d157f51b7a02554005710fbff45c4a6e82]{
 -- Page 77, Offset 38571.
 -- いや、浮気者って、今はそういう場合じゃなくて。
-No, regardless of whether or not I'm a cheater, this isn't the time for that.
+Wait, cheater? That's not what's going on here at all!
 }
 [sha:c741e350fa9052a356ff963ea53edc600763422f]{
 -- Page 78, Offset 38572.
 -- 「部屋から出たのは謝ります、でも誤解ですから！
-"I apologize for coming out of the room, but this is a misunderstanding!
+"I'm sorry about leaving the room,
+ but you've got the wrong idea!
 }
 [sha:73ac0f25713ddcae050bf6b1c7baf49e5763b1fb]{
 -- Page 78, Offset 38573.
 -- どうしても耐えられなくて、外に出たらアルクェイドがいて、アイツ、自分のものになれとか妙なコトを言い出して―――」
-I couldn't handle it anymore, and when I came out, Arcueid was there, and she said something strange asking me to submit to her and stuff―――"
+I couldn't take it anymore,
+ and when I left, I happened to run into Arcueid,
+ and she started saying something weird about how I should become hers―――"
```

Comment:
similar to 37756 (the start is slightly different)

>I just couldn't take it anymore,
 and when I went outside,
 I ran into Arcueid,
 and, like, she told me to become hers and―――"